source/components/organisms/SummaryList/SummaryListItem.tsx

```diff
@@ -105,28 +124,34 @@ const SummaryListItem: React.FC<Props> = ({
             onSelect={changeFromInput}
             color={color}
             style={dateStyle}
+            editable={editable}
           />
         );
       default:
         return (
           <SmallInput
             textAlign="right"
-            value={value}
+            value={value as string}
             onChangeText={changeFromInput}
+            editable={editable}
           />
         );
     }
   };
+  const colorSchema = Object.keys(colorPalette).includes(color) ? color : 'blue';
```

Comment:
Some sort of TODO comment here would be good, since we probably want to replace the color prop in the future. But i do think it's valid to keep the prop at this stage just to not mess around to much (y)

Reply:
Added.